frontend/src/components/TextEditorPopup.tsx

```diff
@@ -122,6 +143,7 @@ const TextEditorPopup = (props: DialogProps) => {
             color="primary"
             variant="contained"
             onClick={handleCompleteClick}
+            sx={{ height: '2rem' }}
```

Comment:
- [ ] Remove this line

Reply:
removed, but it was added to change the button to closer resamble the zeplin one